R/pkg/R/generics.R

```diff
@@ -796,9 +800,17 @@ setGeneric("initcap", function(x) { standardGeneric("initcap") })
 #' @export
 setGeneric("instr", function(y, x) { standardGeneric("instr") })
 
-#' @rdname isNaN
+#' @rdname is.nan
 #' @export
-setGeneric("isNaN", function(x) { standardGeneric("isNaN") })
+setGeneric("is.nan")
```

Comment:
it seems that "is.nan" in base package is masked? There is an implicit generic function for a primitive function. I think no need to define it here.